### DISCUSSION TITLE
Error with pnpm install in wagmi root folder

### DISCUSSION THREAD

**Discussion by Crytogenz** [wevm/wagmi#2188] [Q&A]
I am having a few issues while trying to use wagmi locally. I have cloned the wagmi repo locally and followed the contribution guide here (https://github.com/wagmi-dev/wagmi/blob/main/.github/CONTRIBUTING.md), however on running pnpm install in the wagmi root folder it gives errors

Process followed was:

```
git clone https://github.com/wagmi-dev/wagmi.git --recurse-submodules
cd wagmi
pnpm install
```

Errors are: 
`Failed to create bin at C:\Users\James\Documents\wagmi\examples\_dev\node_modules\.bin\wagmi. The source file at C:\Users\James\Documents\wagmi\packages\cli\dist\cli.js does not exist.`

and

`No projects matched the filters in "C:\Users\James\Documents\wagmi"`

Node version = 18.16.0
pnpm version = 7.27.0
npm version = 9.5.1

**Comment by melroy89** [comment]
yea so I resolved my application my adding a `preinstall` script that first build the project.. and then continues with the pnpm install .. such that the script installs correctly into the `node_module/.bin` directory (of another js project).

This might help other people who also have a pnpm monorepo that tries to export a bin script.
